en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md

```diff
@@ -1,4 +1,4 @@
-# Customizing Task Scheduling
+http://localhost:9000/soap/SimpleStockQuoteService# Customizing Task Scheduling
```

Comment:
Let's remove the change in the title

Reply:
Fixed [e8fadb7](https://github.com/wso2/docs-apim/pull/7325/commits/e8fadb7be21457a13e3e66ecfb9463152a8b0576)